examples/decode.rb

```diff
@@ -0,0 +1,31 @@
+module Authorization
+  #validation using JWT gem
```

Comment:
`require 'jwt' # https://github.com/jwt/ruby-jwt#readme` at the top to make it easier for someone to trace the gem